resources/locale.json

```diff
@@ -745,195 +745,195 @@
-        "UI_PEACOCK_TRADITIONS_JACKIE_HUD": "Eliminate Jackie Carrington",
-        "UI_PEACOCK_TRADITIONS_ACCOMPLICE_HUD": "Eliminate The Accomplice",
-        "UI_PEACOCK_LEEHONG": "The Lee Hong Assassination",
-        "UI_PEACOCK_LEEHONG_DESC": "Good evening 47. Our targets are Lee Hong, the leader of the Red Dragon Triad and Angus Pritchard, a rich antiquities collector but also a former FBI source. They are supposed to meet in a fish restaurant in a few minutes to exchange an ancient gold idol for informations about our client. The client wants you to retrieve these informations. Hong and Pritchard don't want to be seen together so they are eating at seperate tables but will make the exchange in about 10 minutes. This cannot happen 47. Our client also wants you to make believe the exchange went wrong between these two to avoid to be under suspicion. Be creative. I've heard the Triad uses special ammo for their rifles. Maybe you can use this information to increminate the Triad in Pritchard's murder...\nThe clock is ticking 47. Good Luck.\nMade by a Peacock community member known as kercyx.",
-        "UI_PEACOCK_LEEHONG_TRANSACTION": "The Transaction",
-        "UI_PEACOCK_LEEHONG_TRANSACTION_DESC": "Kill Lee Hong and Angus Pritchard to prevent the transaction",
-        "UI_PEACOCK_LEEHONG_BODYGUARDS": "Lee Hong's Bodyguards",
-        "UI_PEACOCK_LEEHONG_BODYGUARDS_DESC": "Neutralize Lee Hong's bodyguards.",
-        "UI_PEACOCK_LEEHONG_BODYGUARDS_HUD": "Neutralize Lee Hong's bodyguards",
-        "UI_PEACOCK_LEEHONG_DOCUMENTS": "The Documents",
-        "UI_PEACOCK_LEEHONG_DOCUMENTS_DESC": "Retrieve the Secret Documents in Pritchard's hotel room.",
-        "UI_PEACOCK_LEEHONG_DOCUMENTS_HUD": "Retrieve the Secret Documents",
-        "UI_PEACOCK_LEEHONG_LEEHONG": "Eliminate Lee Hong",
-        "UI_PEACOCK_LEEHONG_LEEHONG_DESC": "Eliminate Lee Hong.",
-        "UI_PEACOCK_LEEHONG_IDOL": "The Idol",
-        "UI_PEACOCK_LEEHONG_IDOL_DESC": "Retrieve The Golden Idol from Lee Hong's body.",
-        "UI_PEACOCK_LEEHONG_IDOL_HUD": "Retrieve The Golden Idol",
-        "UI_PEACOCK_LEEHONG_RIFLE": "The Gold Dragon",
-        "UI_PEACOCK_LEEHONG_RIFLE_DESC": "Retrieve The Gold Dragon Rifle.",
-        "UI_PEACOCK_LEEHONG_RIFLE_HUD": "Retrieve The Gold Dragon Rifle",
-        "UI_PEACOCK_LEEHONG_ANGUS": "Eliminate Angus Pritchard with The Gold Dragon Rifle",
-        "UI_PEACOCK_LEEHONG_DRIVER": "The Driver",
-        "UI_PEACOCK_LEEHONG_DRIVER_DESC": "Eliminate Lee Hong's driver Blue Lotus in an explosion.",
-        "UI_PEACOCK_LEEHONG_DRIVER_HUD": "Eliminate the Blue Lotus in an explosion",
-        "UI_PEACOCK_POWERED_BY": "Powered by",
+        "UI_PEACOCK_LONGBUSH_GYM_ACCIDENT": "Fitnessstudio Unfall",
+        "UI_PEACOCK_LONGBUSH_GYM_ACCIDENT1_DESC": "Eliminiere Arrush Lal als Personal Trainer verkleidet nachdem du ihn mit einem Trainingsgewicht ruhiggestellt hast",
```

Comment:
"Personal Trainer" sounds like a trainer for staff. "persönlicher Trainer" would be better.